src/stactools/cli/commands/validate.py

```diff
@@ -1,97 +1,62 @@
+import json
 import sys
-from typing import List, Optional
+from typing import Optional
 
 import click
-import pystac
-from pystac import Catalog, Collection, Item, STACObject, STACValidationError
-
-from stactools.core.utils import href_exists
+from stac_validator.validate import StacValidate
 
 
 def create_validate_command(cli: click.Group) -> click.Command:
     @cli.command("validate", short_help="Validate a stac object.")
     @click.argument("href")
     @click.option(
-        "--recurse/--no-recurse",
+        "--recursive/--no-recursive",
+        help="Recursively validate all stac objects in this catalog.",
         default=True,
-        help=(
-            "If false, do not validate any children "
-            "(only useful for Catalogs and Collections"
-        ),
     )
+    @click.option("--links/--no-links", help="Validate links.", default=True)
+    @click.option("--assets/--no-assets", help="Validate assets.", default=True)
+    @click.option("-v", "--verbose", is_flag=True, help="Enables verbose output.")
     @click.option(
-        "--links/--no-links",
-        default=True,
-        help=("If false, do not check any of the objects's links."),
+        "--quiet/--no-quiet", help="Do not print output to console.", default=False
```

Comment:
I think the help needs to always be the positive form, rather than the default behavior?

```suggestion
        "--quiet/--no-quiet", help="Print output to console.", default=False
```